core/posts/serializers.py

```diff
@@ -20,10 +20,19 @@ def to_representation(self, instance):
         instance.author.host = get_host_url()
         instance.author.url = get_author_url(str(instance.author.id))
         representation['author'] = AuthorSummarySerializer(instance.author, read_only=True).data
+        representation['comments'] = CommentsSerializer(instance.comments.all(), many=True, read_only=True).data
```

Comment:
Shouldn't this only be 5 or so comments rather than all of them?

Reply:
Fixed!